kuksa_databroker/doc/timestamp.md

```diff
@@ -0,0 +1,5 @@
+# Databroker Timestamps
+The implementation of KUKSA databroker shall represent the latest value of a ```Datapoint```. Therefore it adds a ```timestamp``` to a ```Datapoint```. If an actuator/provider/application has no system time the databroker sets the ```timestamp``` to the current system time. For some use cases it can be interesting to provide a timestamp set by the actuator/provider/application. For this we added a so called source timestamp (short ```source_ts```) to a ```Datapoint```. This source timestamp is optional and per default set to None.
+
+# Potential attacks
```

Comment:
Shall we possibly call this one something like "Runtime behavior and Potential Attacks"?

And then inside start with something like "KUKSA Databroker always stores only the last Datapoint received for each signal, regardless of what timestamp that is provided in the Datapoint. If an attacker ..."

Shall we possibly also add a link from any of the other *.md documents (like the user guide) to this file